pkg/cli/powertools/forcesetfield/cmd.go

```diff
@@ -53,29 +53,20 @@ func AddCommand(parent *cobra.Command) {
 	options.PopulateDefaults()
 
 	cmd := &cobra.Command{
-		Use:   "force-set-field KIND NAME FIELD.PATH=VALUE",
+		Use:   "force-set-field FIELD.PATH=VALUE",
 		Short: "Sets a field on a KCC object, even immutable fields (experimental)",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			ctx := cmd.Context()
 
-			if len(args) >= 1 {
-				options.Kind = args[0]
-			}
-			if len(args) >= 2 {
-				options.Name = args[1]
-			}
-
 			setFields := map[string]string{}
-			if len(args) >= 3 {
-				for i := 2; i < len(args); i++ {
-					tokens := strings.SplitN(args[i], "=", 2)
-					if len(tokens) < 2 {
-						return fmt.Errorf("expected spec.path=value, got %q", args[i])
-					}
-					k := tokens[0]
-					v := tokens[1]
-					setFields[k] = v
+			for i := 0; i < len(args); i++ {
+				tokens := strings.SplitN(args[i], "=", 2)
+				if len(tokens) < 2 {
```

Comment:
I think you are also in trouble if you get more than 2 tokens? Or are you assuming that the key will never have a '=' so the second '=' must be part of the value?